### PR TITLE
Fix CSV import

### PR DESCRIPTION
**Pull request by Meklo**
Handles BOM encoding
Handles new way lines files are formatted (c.f. [new format explanation](https://odre.opendatasoft.com/explore/dataset/lignes-aeriennes-rte-nv/information/?disjunctive.etat&disjunctive.tension))

### CODE REVIEW

File: src/main/java/org/gridsuite/odre/server/utils/GeographicDataParser.java [hunk 1]
```diff
@@ -177,36 +177,24 @@ private static void parseLine(Map<String, Graph<Coordinate, Object>> graphByLine
             Map<String, String> row;
             while ((row = mapReader.read(headers)) != null) {
                 List<String> ids = Stream.of(row.get(FileValidator.IDS_COLUMNS_NAME.get(FileValidator.CODE_LIGNE_KEY_1)), row.get(FileValidator.IDS_COLUMNS_NAME.get(FileValidator.CODE_LIGNE_KEY_2)), row.get(FileValidator.IDS_COLUMNS_NAME.get(FileValidator.CODE_LIGNE_KEY_3)), row.get(FileValidator.IDS_COLUMNS_NAME.get(FileValidator.CODE_LIGNE_KEY_4)), row.get(FileValidator.IDS_COLUMNS_NAME.get(FileValidator.CODE_LIGNE_KEY_5))).filter(Objects::nonNull).collect(Collectors.toList());
-                if (ids.isEmpty()) {
+                GeoShape geoShape = GeoShapeDeserializer.read(String.valueOf(row.get(FileValidator.GEO_SHAPE)));
```
Comment (ne0ds) [inline src/main/java/org/gridsuite/odre/server/utils/GeographicDataParser.java:180]:
```suggestion
                GeoShape geoShape = GeoShapeDeserializer.read(row.get(FileValidator.GEO_SHAPE));
```

Comment (Meklo) [reply src/main/java/org/gridsuite/odre/server/utils/GeographicDataParser.java]: Done

**Review by ne0ds — APPROVED** [review]
Code OK, test OK

**Review by sBouzols — APPROVED** [review]
Code Review OK
Tests OK

**Pull request merged**